src/Map.js

```diff
@@ -538,6 +538,36 @@ toggleHandlers["showDwellLocations"] = function (enabled) {
   }
 };
 
+/*
+ * Draws circles on the map. Size indicates delta in seconds at that
+ * location.
+ */
+toggleHandlers["showETADeltas"] = function (enabled) {
+  const bubbleName = "showETADeltas";
+  _.forEach(bubbleMap[bubbleName], (bubble) => bubble.setMap(null));
+  delete bubbleMap[bubbleName];
+  const etaDeltas = tripLogs.getETADeltas(minDate, maxDate);
+  if (enabled) {
+    bubbleMap[bubbleName] = _.map(etaDeltas, (etaDelta) => {
+      const circ = new google.maps.Circle({
+        strokeColor: "#000000",
+        strokeOpacity: 0.25,
+        fillColor: "FF0000",
+        fillOpacity: 0.25,
+        map,
+        center: etaDelta.coords,
+        radius: _.min([etaDelta.deltaInSeconds, 300]),
```

Comment:
Why 5m cap?

Reply:
I added some comments to the code.  But it boils down to 'it looks better'.  If the circles get too big then they wash out the whole map.